services/web-server/src/graphql/Hooks.graphql

```diff
@@ -89,11 +95,22 @@ input HookInput {
   triggerSchema: JSON
 }
 
+type HookLastFire {
+  hookGroupId: ID!,
+  hookId: ID!,
+  firedBy: HookFiredBy!,
```

Comment:
Should `firedBy` be required? I don't see it set as required in https://github.com/taskcluster/taskcluster/blob/30372ec731730592a7bf22d4b349a61f75663883/services/hooks/schemas/v1/list-lastFires-response.yml#L41-L47.

If it's not required, then it should be `firedBy: HookFiredBy,`. If it's required, then `list-lastFires-response.yml
` needs to be updated.